### DISCUSSION TITLE
Sudden Disappearance

### DISCUSSION THREAD

**Discussion by Gummomarx** [bijij/ViewImage#244] [Q&A] [answered]
Hello.

Sadly, this important button has vanished from my Firefox 100.0.2 on Windows 7 Ultimate.

Help!

Gummo

**Comment by HaikenEdge** [comment]
Can confirm this is happening on Chrome as well.

**Comment by onmyouji** [comment, marked as answer]
It is a known issue, see here for temporary solution:

https://github.com/bijij/ViewImage/issues/242

**Reply by Gummomarx** [reply]
I used Tampermonkey and installed the script (_a first for me_).  It's a successful fix but I'm hoping for a revised ViewImage as Tampermonkey slows Firefox's initial launch.

**Reply by bijij** [reply]
Hi there. I've just released version 3.6.3 with this fix. Should roll out to webstores today.

**Reply by Gummomarx** [reply]
Thanks but I've just tried it _without success_ on Firefox 101.0 (_Tampermonkey disabled_) .

**Reply by bijij** [reply]
Sorry about that, a 3.6.4 release resolved the issue, there was a second change by google which I hadn't handeled in 3.6.3

**Reply by HaikenEdge** [reply]
When will 3.6.4 be released on Chrome?

**Reply by bijij** [reply]
3.6.4 has been published to the Crome webstore, however google's new manual approval process takes sometimes multiple weeks.
Which is a shame since Mozilla can do the same for Firefox in a matter of minutes.
